server/src/main/java/org/elasticsearch/action/search/SearchResponse.java

```diff
@@ -495,6 +505,7 @@ public static class Clusters implements ToXContentFragment, Writeable {
          * @param remoteClusters number of remote clusters in the search
          * @param ccsMinimizeRoundtrips specifies whether a CCS search is using minimizeRoundtrips feature
          */
+        @Deprecated  // for CCS use the Clusters(OriginalIndices, Map<String, OriginalIndices>, boolean) constructor
```

Comment:
Nothing in the ES code is using this constructor any longer. I'd like to remove it, but since it was marked as public and released in 8.8, I'm assuming I cannot remove it in case any 3rd party modules/plugins are using it?